### PR TITLE
fix(dropdown): reinstated ability to add custom click event listener

### PR DESCRIPTION
**Pull request by alexandereneroth**
Fixes: [AB#2603](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2603)

Reinstated ability to add custom click event listener

**How to test**  
Check netlify deploy preview

### CODE REVIEW

**Comment by mJarsater** [comment]
We would need to add the changes to the tegel folder as well.

**Comment by alexandereneroth** [comment]
> We would need to add the changes to the tegel folder as well.

Good call, ill get on that

**Comment by mJarsater** [comment]
> > We would need to add the changes to the tegel folder as well.
> 
> Good call, ill get on that

I guess the build issue is due to us not having a tagel folder in the branch.

**Comment by alexandereneroth** [comment]
I accidentally formatted some unrelated files when i rebased, thats why there are changes in textfield for example. Should just be formatting

File: components/src/components/dropdown/dropdown-option.tsx [hunk 1]
```diff
@@ -54,10 +57,7 @@ export class DropdownOption {
       });
     }
   }
-  @Listen('click')
-  handleClick(ev) {
-    ev.stopPropagation();
-  }
+
```
Comment (mJarsater) [inline components/src/components/dropdown/dropdown-option.tsx:60]:
```suggestion
  @Listen('click')
  handleSelection(ev) {
    if(this.isMultiSelectOption){
      ev.stopPropagation()
    }
  }
```
Maybe just add it on the multiselect for now?

**Comment by theJohnnyMe** [comment]
We need to double-check if we caused this, it was there, or was it introduced just now? 
It seems to be possible to select more then one option in default dropdown 👀 

Screenshot 2022-11-21 at 11 52 54

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Pull request merged**